### PR TITLE
Added urdu language support

### PR DESCRIPTION
**Pull request by shah78677**
Added Urdu Language support. Solves issue #85

### CODE REVIEW

**Comment by shah78677** [comment]
Kindly add "hacktoberfest-accepted" label to this PR so that it counts towards my hacktoberfest PRs. Thanks.

File: src/main.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ register("pt", () => import(/* webpackChunkName: "pt" */ "./lang/pt.json"));
 register("ru", () => import(/* webpackChunkName: "ru" */ "./lang/ru.json"));
 register("tr", () => import(/* webpackChunkName: "tr" */ "./lang/tr.json"));
 register("ja", () => import(/* webpackChunkName: "ja" */ "./lang/ja.json"));
+register("ur", () => import(/* webpackChunkName: "ja" */ "./lang/ur.json"));
```
Comment (TorstenDittmann) [inline src/main.js:20]: Can you modify `/* webpackChunkName: "ja" */` to `/* webpackChunkName: "ur" */` as well? 🙂

Comment (shah78677) [reply src/main.js:20]: Oh, sorry I missed that. Will send a fix.

Comment (shah78677) [reply src/main.js]: I have corrected the code. Kindly review.

**Pull request merged**